sdk/lambdatesthyperexecute/azure-mgmt-lambdatesthyperexecute/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0b1 (2025-04-16)
```

Comment:
@aggarwalsw please remember to update date to latest when you are to merge/release.